haas_websocket/main.py

```diff
@@ -26,18 +26,9 @@ def filterMessage(message,publisher):
     
     data_type = data.get('type')
     # Filters the messages and publishes messages to specified topic
-    if data_type == 'point':
-        path = publisher.topic_path(project_id, os.getenv('POINT_TOPIC'))
-        returnMessage = 'point'
-    elif data_type == 'heartbeat':
-        path = publisher.topic_path(project_id, os.getenv('HEARTBEAT_TOPIC'))
-        returnMessage = 'heartbeat'
-    elif data_type == 'thing':
-        path = publisher.topic_path(project_id, os.getenv('THING_TOPIC'))
-        returnMessage = 'thing'
-    elif data_type == 'location':
-        path = publisher.topic_path(project_id, os.getenv('LOCATION_TOPIC'))
-        returnMessage = 'location'
+    if data_type == 'point' or data_type == 'heartbeat' or data_type == 'thing' or data_type == 'location':
+        path = publisher.topic_path(project_id, 'haas-alert-raw')
```

Comment:
This is just all going to raw topic now, and we standardize down the pipe?

Reply:
Yeah, this is the pipeline that Stefan outlined for the data-flow: https://lucid.app/lucidchart/2851cc5b-abbd-401d-898c-2894c076b5f9/edit?invitationId=inv_e3af46bb-bbd8-411d-84f7-375e902d48dd&page=0_0#
let me know if you can't access the file